1-create-unprivileged-user.sh

```diff
@@ -19,17 +20,32 @@ data:
   htpasswd: ${htpwd_encoded}
 EOF
 
-kubectl patch oauth cluster --type='json' -p '[
-  {
-    "op":"add","path":"/spec/identityProviders/1",
-    "value":{
-      "name":"htpasswd",
-      "mappingMethod":"claim",
-      "type":"HTPasswd",
-      "htpasswd":
-        {"fileData":
-          {"name":"htpass-secret"}
-        }
+# if there are no indentityProviders
+if [[ $(kubectl get oauth cluster -o=jsonpath='{.spec.identityProviders[*].name}' | wc -c) -eq 0 ]]; then
```

Comment:
If statements are used to check if the identityProvider should be added with `kubectl patch`, since this script (`1-create-unprivileged-user.sh`) is ran multiple times to add users:

https://github.com/redhat-developer/devspaces-demo/blob/a0cd446744978f9503dcd85804ac061fe014d8aa/.devfile.yaml#L29-L33